client/cypress/model/SignInPage.js

```diff
@@ -0,0 +1,44 @@
+class SignInPage {
+  // Sign in form
+  signInEmailField = "//h2[contains(text(),'I already')]/following-sibling::form//input[@type=\"email\"]";
+  signInPasswordField = "//h2[contains(text(),'I already')]/following-sibling::form//input[@type=\"password\"]";
+  signInSignInButton = "//h2[contains(text(),'I already')]/following-sibling::form//button[@type=\"submit\"]";
+
+  //Sign Up form
+  signUpNameField = "//h2[contains(text(),'Create account')]/following-sibling::form//input[@name=\"displayName\"]";
+  signUpEmailField = "//h2[contains(text(),'Create account')]/following-sibling::form//input[@type=\"email\"]";
+  signUpPasswordField = "//h2[contains(text(),'Create account')]/following-sibling::form//input[@name=\"password\"]";
+  signUpConfirmPasswordField = "//h2[contains(text(),'Create account')]/following-sibling::form//input[@name=\"confirmPassword\"]";
+  signUpSignUpButton = "//h2[contains(text(),'Create account')]/following-sibling::form//button";
+
+  // Fill in Sign In form
+  fillSignInForm(email, password){
+    this.insertField(this.signInEmailField, email);
+    this.insertField(this.signInPasswordField, password);
+
+    return this;
+  }
+
+  // Fill in Sign Up form
+  fillSignUpForm(name, email, password, confirmPassword){
+    this.insertField(this.signUpNameField, name);
+    this.insertField(this.signUpEmailField, email);
+    this.insertField(this.signUpPasswordField, password);
+    this.insertField(this.signUpConfirmPasswordField, confirmPassword);
+  }
```

Comment:
you do not have test for this method, why are you even added this?

do not add redundant code unless you are working on it.